apps/roam/src/utils/settings.ts

```diff
@@ -0,0 +1,34 @@
+import getExtensionAPI from "roamjs-components/util/extensionApiContext";
```

Comment:
nit: can we rename this file to something that is less generic?  `settings` could refer to a number of things.
some ideas: 
- extensionSettings
- roamExtensionConfig
- settingsManager